docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+* Click **Review+Create**
+* Click **Create**
+* Click **Go to resource** (option available after Cosmos DB account creation)
+* Select **Default consistency** from left navigation
+    * Select _STRONG_ and click **Save**
+
+## Create Bastion
+
+You must have a bastion server to access the private AKS cluster. You can use the following steps to create a bastion server.
+ 
+* Select [Virtual machines](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines) from Azure portal
+* Click **Create**
+* Select **Virtual machine**
+* On the Basics page, configure the following options
+    * Select an Azure **Subscription**
+    * Select your **Resource group**
+    * Enter **Virtual machine name**
+    * Select **Region**
+    * Select any of the following availability options in the dropdown menu, select any of the following availability options in the dropdown menu, You can select the _No infrastructure redundancy required_ option if you don't want any.
+        * Availability set: Select created availability set from dropdown menu
+        * Availability zone: Select an availability zone 1 or 2 or 3
+    * Select Image _CentOS-based 7.9 - Gen 1_
+    * Select **Size** as _Standard_B1s_ from dropdown menu
+    * Select authentication type as SSH public key(generate an SSH key to access the server) or password(can use username and password to SSH access the server).
+    * Enter **Username** as centos
+    * If you are using **Password** for authentication type.
+        * Enter **Password**
+        * Enter **Confirm password**
+    * If you are using **SSH public key** for authentication type.
+        * Select **Generate new key pair**
+        * Select **Use existing public key** and upload already generated public key in SSH public key box.
+    * Select **Public inbound ports** as _Allow selected ports_.
+    * **Select inbound ports** as _SSH (22)_ from the dropdown menu.
+* On the Networking page, select virtual network settings
+    * Select your **virtual network** from drop-down menu
+    * Select subnet **public**
+    * Click **Review+Create**
+    * Click **Create**
+
+## Create Kubernetes Cluster
+
+You can use the following steps to configure Kubernetes cluster based on the [requirements](./ManualDeploymentGuideScalarDLOnAzure.md#step-3-configure-aks)
+
+* Select [Kubernetes services](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.ContainerService%2FmanagedClusters) from Azure portal
+* Click **Create**
+* Select **Create a Kubernetes cluster**
+* On the Basics page, configure the following options
+    * Select your **Subscription**
+    * Select your **Resource group**
+    * Enter **Kubernetes Cluster name**
+    * Select **Region**
+    * Select **Availability zones** from drop down if you want to create in multiple availability zones
+    * Select **Kubernetes version** _1.19.11_ from dropdown
```

Comment:
I think it's better not to specify the patch version. `1.19.x` ?

Reply:
The current document does not specify the k8s patch version.